addon/components/column-chart/component.js

```diff
@@ -286,6 +286,13 @@ export default BaseChartComponent.extend({
             return;
         }
 
+        chart.selectAll('rect.bar').each(function () {
+            const bar = d3.select(this);
+            if (bar.attr('height') !== '0') {
+                bar.attr('tabindex', 0);
+            }
+        });
```

Comment:
maybe we can add this attribute in a place where we are already selecting all of the bars?

Reply:
Done.